src/js/datepicker.js

```diff
@@ -16,52 +16,17 @@ DatePicker = class {
     id,
     numberOfMonthsInFuture = 12,
     numberOfMonthsInPast = 12,
-    fromYear = 2019,
-    fromMonth = 0,
-    tillYear = 2020,
-    tillMonth = 11,
+    // fromYear = 2019,
```

Comment:
i guess at it's infinite these arent needed anymore right? Long term you could resuse that as options for a component to limit the range of the months

Reply:
removed these for now :)